deployment/GI05MOAS-GL538_Deploy.csv

```diff
@@ -2,4 +2,8 @@ CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentN
 AR69-01,,AJ20230507,,GI05MOAS-GL538-00-ENG000000,2,1,2022-06-28T17:27:00,2023-05-09T08:56:00,CGVGI-05MOAS-00000,CGVEH-GLDROO-00538,CGCON-GLDROO-00538,59.81694,-39.11426,,1000,1000,,,
 AR69-01,,AJ20230507,,GI05MOAS-GL538-01-FLORDM000,2,1,2022-06-28T17:27:00,2023-05-09T08:56:00,CGVGI-05MOAS-00000,CGVEH-GLDROO-00538,CGINS-FLORDM-03063,59.81694,-39.11426,,1000,1000,,,
 AR69-01,,AJ20230507,,GI05MOAS-GL538-02-DOSTAM000,2,1,2022-06-28T17:27:00,2023-05-09T08:56:00,CGVGI-05MOAS-00000,CGVEH-GLDROO-00538,CGINS-DOSTAM-00438,59.81694,-39.11426,,1000,1000,,,
-AR69-01,,AJ20230507,,GI05MOAS-GL538-04-CTDGVM000,2,1,2022-06-28T17:27:00,2023-05-09T08:56:00,CGVGI-05MOAS-00000,CGVEH-GLDROO-00538,CGINS-CTDGVM-00104,59.81694,-39.11426,,1000,1000,,,
\ No newline at end of file
+AR69-01,,AJ20230507,,GI05MOAS-GL538-04-CTDGVM000,2,1,2022-06-28T17:27:00,2023-05-09T08:56:00,CGVGI-05MOAS-00000,CGVEH-GLDROO-00538,CGINS-CTDGVM-00104,59.81694,-39.11426,,1000,1000,,,
+AR84-01,,,,GI05MOAS-GL538-00-ENG000000,3,1,2024-06-12T10:05:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00538,CGCON-GLDROO-00538,59.906,-39.2766,,1000,1000,,,
+AR84-01,,,,GI05MOAS-GL538-01-FLORDM000,3,1,2024-06-12T10:05:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00538,CGINS-FLORDM-03063,59.906,-39.2766,,1000,1000,,,
+AR84-01,,,,GI05MOAS-GL538-02-DOSTAM000,3,1,2024-06-12T10:05:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00538,CGINS-DOSTAM-00438,59.906,-39.2766,,1000,1000,top mount,,
+AR84-01,,,,GI05MOAS-GL538-04-CTDGVM000,3,1,2024-06-12T10:05:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00538,CGINS-CTDGVM-00104,59.906,-39.2766,,1000,1000,,,
```

Comment:
looks good

Reply:
Looks good.  Assume glider instrument S/Ns are supposed to be the same as the previous deployment.